@here/harp-mapview/test/TextElementsRendererTestFixture.ts

```diff
@@ -316,6 +340,24 @@ export class TestFixture {
         return true;
     }
 
+    private setTerrainTiles(indices: number[]) {
+        expect(this.m_viewState.elevationProvider).not.undefined;
+        this.m_elevationProviderStub.getDisplacementMap.resetBehavior();
+        this.m_elevationProviderStub.getDisplacementMap.returns(undefined);
+
+        for (const index of indices) {
+            const tileKey = this.m_allTiles[index].tileKey;
+            this.m_elevationProviderStub.getDisplacementMap
```

Comment:
do the arguments matter? Why not just .getDisplacementMap.returns(...) ?

Reply:
the test is simulating how the terrain tiles become available in the different frames.
At the beginning of a frame, that function "terrainTiles()" is called with the indices of the terrain tiles that should be available at that moment. For those available, getDisplacementMap must return the displacement map. For any other tiles that function returns undefined, as it's defined just above that loop.

Reply:
ok, makes sense.